foundation-extensions/lifecycle-for-edge-network/README.md

```diff
@@ -0,0 +1,93 @@
+# Lifecycle for Edge Network
+
+The Adobe Experience Platform Lifecycle mobile extension enables application lifecycle data collection from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/).
+
+
```

Comment:
Can we add something like 
Follow these steps to forward lifecycle data to Platform.